src/Review.js

```diff
@@ -0,0 +1,38 @@
+import { Box, Button, Card, CardBody, CardFooter, CardHeader, Text } from "grommet";
+import { Edit, Favorite, Trash } from 'grommet-icons';
+
+export default function Review(props) {
```

Comment:
ここで展開してしまえば `props.xxx` みたいにしなくて済みそうです

```suggestion
export default function Review({ review }) {
```